crates/services/txpool_v2/src/tests/tests_pool.rs

```diff
@@ -1259,6 +1259,35 @@ fn insert_tx_with_blob_already_insert_at_lower_tip() {
     universe.assert_pool_integrity(&[result.unwrap().0]);
 }
 
+#[test]
+fn insert__tx_depends_on_extracted_tx() {
```

Comment:
nit: this name doesn't tell me what I'm expecting. 

perhaps `verify_and_insert__when_dependent_tx_is_extracted_new_tx_still_accepted` or something like that.